theme6/source/stylesheets/main.css

```diff
@@ -0,0 +1,280 @@
+  justify-content: flex-end;
+  height: 300px;
+  text-align: center;
+}
+
+.mid-banner-saylor {
+  background-image: url("../images/mid_banner1.png");
+  background-size: 130%;
+  background-repeat: no-repeat;
+  background-position: top;
+}
+
+.mid-banner-archeologist {
+  background-image: url("../images/mid_banner2.png");
+  background-size: 130%;
+  background-repeat: no-repeat;
+  background-position: top;
+}
+
+.mid-banner-saylor .mid-banner-content {
+  background: linear-gradient(0deg,
+    rgba(239, 239, 239),
+    rgba(239, 239, 239, 1) 20%,
+    rgba(239, 239, 239, 0.2) 50%,
+    rgba(239, 239, 239, 0));
+  color: #000000;
+}
+
+.mid-banner-archeologist .mid-banner-content {
+  background: linear-gradient(0deg,
+    rgba(189, 152, 107),
+    rgba(189, 152, 107, 1) 10%,
+    rgba(189, 152, 107, 0.2) 50%,
+    rgba(189, 152, 107, 0));
+  color: #FFFFFF;
+}
+
+.mid-banner-content h3 {
+  margin-bottom: 0;
+}
+
+.mid-banner-content {
+  font-family: 'Lato', sans-serif;
+}
+
+.mid-banner-content button {
+  font-family: 'Lato', sans-serif;
+  background-color: Transparent;
+  border-radius: 2px;
+  border-style: solid;
+  margin-bottom: 12px;
+  margin-left: 5px;
+  margin-right: 5px;
+}
+
+.mid-banner-saylor button {
+  border-color: #627387;
+}
+
+.mid-banner-archeologist button {
+  border-color: #FFFFFF;
+}
+
+.mid-banner-saylor button h1 {
+  margin: 0;
+  padding: 10px 0 10px 0;
+  color: #627387;
+}
+
+.mid-banner-archeologist button h1 {
+  margin: 0;
+  padding: 10px 0 10px 0;
+  color: #FFFFFF;
+}
+
+@media only screen and (width: 768px) {
+  .big-banner {
+    background-size: cover;
+    background-position: center;
+  }
+
+  .big-banner-content p {
+    margin-bottom: 20px;
+  }
+
+  .small-banner {
+    background-size: cover;
+    background-position: bottom;
+  }
+
+  .mid-banner-saylor {
+    background-size: cover;
+    background-position: center;
+  }
+
+  .mid-banner-archeologist {
+    background-position: top;
+  }
+}
+
+@media only screen and (min-width: 769px) {
+  .flex-grid {
+    display: flex;
+    margin-bottom: 20px;
+  }
+
+  .big-banner {
+    flex: 2;
+    display: flex;
+    background-image: url("../images/banner1.png");
+    background-size: contain;
+    background-position: left;
+    justify-content: flex-end;
+    height: 400px;
+    margin-right: 20px;
+  }
+
+  .big-banner-content {
+    display: flex;
+    background: linear-gradient(270deg,
+      rgba(251,239,227),
+      rgba(251,239,227, 1) 20%,
+      rgba(251,239,227, 1) 60%,
+      rgba(251,239,227, 0));
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    height: 400px;
+    width: 50%;
+  }
+
+  .big-banner-content p {
+    padding-left: 20px;
+    padding-right: 20px;
+    font-size: 20px;
+    margin-top: 20px;
+    margin-bottom: 30px;
+  }
+
+  .big-banner-content h1 {
+    font-size: 32px;
+  }
+
+  .big-banner-content button {
+    padding-left: 50px;
+    padding-right: 50px;
```

Comment:
Just a quick tip, no need to make changes, but you can also add this to one line. `padding: 0 50px;`